controllers/dscinitialization/dscinitialization_test.go

```diff
@@ -199,7 +199,7 @@ var _ = Describe("DataScienceCluster initialization", func() {
 	})
 })
 
-// cleanup utility func
+// cleanup utility func.
```

Comment:
```suggestion
```

I have a simple rule that comments should rather state the non-obvious things :) This I can read from the name itself. And since it's unexported func, we can live without docs